.github/workflows/rhub.yaml

```diff
@@ -0,0 +1,108 @@
+# R-hub's generic GitHub Actions workflow file. Its canonical location is at
+# https://github.com/r-hub/actions/blob/main/workflows/rhub.yaml
+
+name: R-hub 🌐
+
+on:
+  workflow_call:
+    inputs:
+      config:
+        description: 'A comma separated list of R-hub platforms to use.'
+        type: string
+        # These platforms correspond roughly to CRAN package checks.
+        # More information: https://github.com/insightsengineering/idr-tasks/issues/781
+        default: 'ubuntu-clang,ubuntu-gcc12,gcc13,ubuntu-next,ubuntu-release,macos-arm64,macos,windows'
```

Comment:
To be decided for which platforms the R-hub check should be run by default (since this is an input, of course it is customizable by the calling workflow).
We might want include some more Fedora-based checks: `gcc14, atlas, valgrind, intel, nosuggests, mkl`.